en/admin-guide/db-customization/index.rst

```diff
@@ -0,0 +1,69 @@
+Customization of the data model
+===============================
+
+This chapter describes the possible customization to the data model.
+
+
+Introduction
+^^^^^^^^^^^^^
+
+The data model is versionned and updates are achieved using Postgres Updates Manager (aka `PUM <https://github.com/opengisch/pum>`_).
+The views required to edit the data are automatically generated using `Pirogue <https://github.com/opengisch/pirogue>`_ library.
+
+
+The combination of both allows to introduce small customizations of the data model such as adding fields to existing base tables or extending views by joining additional tables.
+
+
+This chapter describes these possible customizations.
+
+
+Adding fields
+^^^^^^^^^^^^^
+
+It is allowed to add additional fields to tables by naming them as ``XXX` (TO BE DEFINED).
```

Comment:
To be defined?

Reply:
we should define a way of naming field (with `usr_*` or whatever)

Reply:
I think we should go to usr_* like QWAT. @sjib thoughts?